tests/functional/aws-node-sdk/test/bucket/putBucketReplication.js

```diff
@@ -175,18 +177,16 @@ describe('aws-node-sdk test putBucketReplication configuration rules', () => {
             },
         });
         config.Role = ARN;
-
-        it('should accept configuration if \'Role\' is a single valid ' +
-            `Amazon Resource Name: '${ARN}', and a rule storageClass defines ` +
-            'an external location', done =>
-            checkError(config, null, done));
+        const test = `should allow only one role to be specified for external
+         locations`;
+        itSkipIfE2E(test, done => checkError(config, null, done));
     });
 
     it('should allow a combination of storageClasses across rules', done => {
         const config = setConfigRules([replicationConfig.Rules[0], {
             Destination: {
                 Bucket: `arn:aws:s3:::${destinationBucket}`,
-                StorageClass: 'us-east-2',
+                StorageClass: 'us-east-1',
```

Comment:
not sure if it's just flakiness but it looks like this change makes this test fail [here on CI](https://eve.devsca.com/github/scality/cloudserver/#/builders/17/builds/2907/steps/8/logs/stdio) somehow

Reply:
Hmm, I assumed that this change would be safe as all it is testing it combining two different storage classes for validation, may be not.

Reply:
Yeah, it's a bit unforeseen. :)